app/controllers/companies_controller.rb

```diff
@@ -11,15 +12,17 @@ def create
     @company = Company.new(company_params)
     if @company.save
       flash[:success] = "#{@company.name} added!"
-      redirect_to company_path(@company)
+      redirect_to companies_path
     else
       render :new
     end
   end
 
   def show
-    company = Company.find(params[:id])
-    redirect_to company_jobs_path(company)
+    @company = Company.find(params[:id])
+    @contact = Contact.new()
+    @contact.company_id = @company.id
```

Comment:
again, this isn't necessary since we aren't actually creating the `Contact` here